src/Skia/Perspex.Skia/FormattedTextImpl.cs

```diff
@@ -75,36 +108,206 @@ public Size Measure()
 
         public void SetForegroundBrush(IBrush brush, int startIndex, int length)
         {
+            // Is this method even relevant now???
```

Comment:
Yah this was not implemented in the previous skia backend either. So I think for now this should not hold up this initial giant merge. We can make the formatted text feature a separate issue.